src/python/pants/option/global_options.py

```diff
@@ -227,6 +227,51 @@ def disabled(cls) -> DynamicRemoteOptions:
             execution_rpc_concurrency=DEFAULT_EXECUTION_OPTIONS.remote_execution_rpc_concurrency,
         )
 
+    @classmethod
+    def _use_oauth_token(cls, bootstrap_options: OptionValueContainer) -> DynamicRemoteOptions:
+        oauth_token = (
+            Path(bootstrap_options.remote_oauth_bearer_token_path).resolve().read_text().strip()
+        )
+        if set(oauth_token).intersection({"\n", "\r"}):
```

Comment:
When is there a need to check for `'\r'`? Why not just `if "\n" in oauth_token`?

Reply:
idk. this code was like this before. I don't thins we need to change this particular check in this PR.
it was added here:
https://github.com/pantsbuild/pants/commit/2a5346e7937b43503a3e35b248da5248b6015ff3#diff-db4651ae839e702898bac44a00db5fd93b21701f624f57479fb537d7bc55a6d1R111
so maybe @Eric-Arellano can provide some context on it.

Reply:
I think it was added a long time ago from Daniel Wagner Hall. We should probably keep it.